examples/reminderbot/credentials.yml

```diff
@@ -20,11 +21,6 @@ callback:
 #  slack_token: "<your slack token>"
 #  slack_channel: "<the slack channel>"
 
-#socketio:
-#  user_message_evt: <event name for user message>
-#  bot_message_evt: <event name for bot messages>
-#  session_persistence: <true/false>
-
```

Comment:
why was this removed?

Reply:
arg sorry this was me trying to narrow down my issues

Reply:
I uncommented them, then deleted them